interp/notation.ml

```diff
@@ -760,16 +760,20 @@ let rebuild_arguments_scope (req,r,n,l,_) =
   match req with
     | ArgsScopeNoDischarge -> assert false
     | ArgsScopeAuto ->
-        let scs,cls = compute_arguments_scope_full (fst(Global.type_of_global_in_context (Global.env ()) r)(*FIXME?*)) in
-	(req,r,List.length scs,scs,cls)
+      let sigma, env = Evd.empty, Global.env () in (*FIXME?*)
+      let typ = EConstr.of_constr @@ fst (Global.type_of_global_in_context env r) in
+      let scs,cls = compute_arguments_scope_full sigma typ in
+      (req,r,List.length scs,scs,cls)
```

Comment:
It seems like we only use the evar map to pass it to `Reductionops.whd_betaiotazeta` as that works on econstr only. Shouldn't we instead define some `Reductionops.NoEvars.whd_...` which would operate on constr and take no evar map?
Also I think the FIXME is about discarding the universe context from `type_of_global`.